scripts/linters/custom_eslint_checks/rules/protractor-practices.spec.js

```diff
@@ -67,6 +67,12 @@ ruleTester.run('protractor-practices', rule, {
     code: 'var modal = element.all(by.css(".modal-dialog")).last();'
   }, {
     code: 'var modal = element(by.css("option:checked"));'
+  }, {
+    code: '// Filter results from the array'
```

Comment:
To make clearer that we're testing the lint rule:

```suggestion
    code: '// Filter results from the array using arr.filter()'
```